example/lib/pages/player/rotation/human_topdown_player.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:bonfire/bonfire.dart';
+import 'package:example/pages/player/rotation/human_topdown_player_spritesheet.dart';
+
+class HumanTopdownPlayer extends RotationPlayer with BlockMovementCollision {
+  HumanTopdownPlayer({
+    required Vector2 position,
+  }) : super(
+          position: position,
+          size: Vector2(24, 16),
+          animIdle: HumanTopdownPlayerSpritesheet.idle(),
+          animRun: HumanTopdownPlayerSpritesheet.run(),
+          speed: 32,
+        );
+
+  @override
+  Future<void> onLoad() {
+    add(CircleHitbox(radius: 8, position: Vector2(4, 0)));
```

Comment:
Now that the map has collisions, I could manage to get stuck on the walls, maybe it's because the shape difference?

Reply:
I think not. When I centering this not happened more.
It's happen because my mixin `BlockMovementCollision` not consider rotation to block.